pyproject.toml

```diff
@@ -4,15 +4,15 @@ version = "0.1.0"
 description = ""
 authors = []
 license = "MIT"
+#package.mode = false
 
 [tool.poetry.dependencies]
 python = "^3.10"
```

Comment:
Has torch been updated to support Python 3.12 yet?

Reply:
Simulation says they would be held back cause of airsim hasn't been upgraded to 3.10, but pytorch has at least partial if not mostly/full implementation on 3.12

Reply:
Alright, so we don't need to update Python until AirSim supports 3.10 then.